script.js

```diff
@@ -0,0 +1,123 @@
+const theForm = document.querySelector(".formContainer")
```

Comment:
Make sure you have prettier installed and it is configured for JavaScript files.